solutions/observability/synthetics/network-security.md

```diff
@@ -43,16 +44,13 @@ Note that as regions are added, this list will change. Similarly existing region
 
 If you’re running tests from [{{private-location}}s](/solutions/observability/synthetics/monitor-resources-on-private-networks.md), you will have the {{agent}} installed on host machines that run the tests. You need to obtain the address ranges for these machines. This needs to be the IP address that the host is making the connection from into the {{es}} cluster. This *might not* be the IP address bound to the network interface of the host machine, but the proxy or other address based on your network configuration.
 
-## Add the traffic filter [_add_the_traffic_filter]
+## Add the IP filter [_add_the_traffic_filter]
 
-Once you know the CIDR blocks for your testing sources, add them to your {{es}} deployment. Find detailed instructions in the [IP traffic filters](/deploy-manage/security/ip-traffic-filtering.md) docs.
+Once you know the CIDR blocks for your testing sources, add them to your {{es}} deployment. Find detailed instructions in the [](/deploy-manage/security/ip-filtering.md) docs.
```

Comment:
```suggestion
Once you know the CIDR blocks for your testing sources, add them to your {{es}} deployment. Find detailed instructions in the [IP filtering](/deploy-manage/security/ip-filtering.md) docs.
```

Reply:
these empty titles automatically populate the page title, which is a slight maintainability win